django_ninja_crudl/endpoints/update.py

```
@@ -98,17 +91,34 @@
-                        m2m_manager.set(attr_value)
-                    else:
-                        raise
-            obj.save()
+            simple_fields, relational_fields = self._get_fields_to_set(
+                config.model, payload
+            )
+
+            def update() -> None:
+                nonlocal obj
+                with validating_manager(config.model):  # noqa: WPS220
+                    # TODO(phuongfi91): should we use validating_manager later as well?
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)